### ISSUE TITLE
GSL 2.4 is out...

### ISSUE THREAD

**Issue by tschoonj** [reinh-bader/fgsl#21]
and introduces a couple of new features which will need support in FGSL...

**Comment by reinh-bader** [comment]
It will be a bit before I can start working on this, due to currently high workload.
Cheers

**Comment by reinh-bader** [comment]
I've released 1.3.0. There are some issues with the infrastructure I was not able to resolve:
(1) Travis CI appears to be broken because it cannot fetch the gsl-2.4 archive needed as prerequisite
(2) I've seen some (minor) failures in the .log files that were not reflected in the .trs files. Apparently the test-driver does not properly transmit the information, but I've not been able to figure out how this is done, much less fix the problem.
Manual builds and tests for a couple of compilers worked fine, so I went ahead anyway. I hope to get around 2.5 and 2.6 support soon.
Cheers
Reinhold
